projects/examples/src/components/form-input/number-with-unit-form-input.example.component.ts

```diff
@@ -67,9 +67,5 @@ export class NumberWithUnitFormInputExampleComponent implements AfterViewInit, O
         });
     }
 
-    ngAfterViewInit(): void {
-        this.formGroup.controls.cpuLimit.setValue(-1);
```

Comment:
I think there is a case to set the value later and not as part of the formGroup creation. But it is pitty there is no comment (may be it was me that did it and missed to add a comment).
Just double check please

Reply:
Ok. I'm going to merge this because Bryan needs the fixes. But we can talk about this on Monday in the parking lot.